gui/default/syncthing/core/syncthingController.js

```diff
@@ -831,7 +841,7 @@ angular.module('syncthing.core')
             if (status === 'unknown') {
                 return 'info';
             }
-            if (status === 'stopped' || status === 'outofsync' || status === 'error' || status === 'faileditems') {
+            if (status === 'stopped' || status === 'outofsync' || status === 'error' || status === 'faileditems' || status === 'localunencrypted') {
```

Comment:
Again, it's not really localunencrypted, it's more like unexpected, which could be localadditions?

If we treat this (localadditions) as success for receiveonly, why are we treating this as a error for receiveencrypted?

Reply:
I didn't notice local additions was treated like a success - I don't see how that makes sense and if anything, that should be changed.